knowledge/tech_industry/rhacs/attribution.txt

```diff
@@ -0,0 +1,4 @@
+Title of work: RHACS documentation
+Link to work: https://github.com/jalvarez-rh/instruct-acs-docs/blob/main/acs-4.4-docs.md
```

Comment:
I'm sorry but we are only taking Wikipedia submissions right now. Until we have clear direction from legal please put a PR with this domain in this: https://github.com/instructlab/dev-docs/blob/main/docs/knowledge-submissions-past-wikipedia.md